src/config.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
@brian-frederick Does React have a good way to provide a local `.env` file of some kind? Perhaps this: https://stackoverflow.com/a/49579700/1215749

Reply:
Realizing after writing this that none of these are keys so you might already be doing this.

Reply:
@whob13 
using a .env file would definitely be a nice improvement. Love that and we should do that for sure. That said, none of this stuff is secret and all of it will be exposed in the browser, just so you know.